WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewController+Activity.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+// MARK: - SearchableActivity Conformance
+extension InPersonPaymentsMenuViewController: SearchableActivityConvertable {
+    var activityType: String {
+        return WooActivityType.payments.rawValue
+    }
+
+    var activityTitle: String {
+        return NSLocalizedString("Payments", comment: "Title of the 'Payments' screen - used for spotlight indexing on iOS.")
+    }
+
+    var activityDescription: String? {
+        return NSLocalizedString("Collect payments, setup Tap to Pay, order card readers and more.",
+                                 comment: "Description of the 'Payments' screen - used for spotlight indexing on iOS.")
+    }
+
+    var activityKeywords: Set<String>? {
+        let keyWordString = NSLocalizedString("payments, tap to pay, woocommerce, woo, in-person payments, in person payments" +
+                                              "collect payment, payments, reader, card reader, order card reader",
+                                              comment: "This is a comma separated list of keywords used for spotlight indexing of the 'Payments' screen.")
+        let keywordArray = keyWordString.arrayOfTags()
+
+        guard !keywordArray.isEmpty else {
+            return nil
+        }
+
+        return Set(keywordArray)
```

Comment:
We're duplicating this bit each time we need to implement `activityKeywords`, perhaps we could adapt `arrayOfTags()` to be a sort of `setOfTags()` instead?

Reply:
Indeed, done in 739f5a169b6c3607cfa1f7da99e5579458cd5f39